src/test/java/com/hotels/mutantswarm/report/ReportTest.java

```diff
@@ -102,7 +101,6 @@ public void testingReport() throws IOException {
     when(line1.getMutationCount()).thenReturn(0);
     when(line1.getKilledCount()).thenReturn(0);
     when(line1.getNumber()).thenReturn(0);
-    when(line1.getSurvivorCount()).thenReturn(0);
```

Comment:
Are the lines that were removed not needed?

Reply:
They're not. In the new version of mockito, it 'forces' you to delete the unused mocks; it actually tells you which ones they are 👍 So I took them all out and tested everything again to make sure it worked

Reply:
ooh thats handy 👌🏻